hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -161,8 +161,10 @@ def react_to_threat_exchange(
     Puts a ThreatExchangeReactionMessage on the queue to be processed asynchronously
     """
     logger.info(
-        f"The contents of a ThreatExchangeReactionMessage will contain match_message = {match_message} and threat_exchange_reaction_label = {threat_exchange_reaction_label}"
+        "The contents of a ThreatExchangeReactionMessage will contain the following:"
     )
+    logger.ingo("match_message = %s", match_message)
+    logger.info("threat_exchange_reaction_label = %s", threat_exchange_reaction_label)
```

Comment:
A potential downside of this approach is that if your log format adds a prefix (such as timestamp), it may make it hard to see these are all part of one message, since it will be interleveled with logs from other threads.